docs/docs-manifest.json

```diff
@@ -1498,6 +1498,19 @@
         }
       ]
     },
+    {
+      "category_slug": "release",
+      "category_title": "Release",
+      "posts": [
+        {
+          "edit_url": "https://github.com/woocommerce/woocommerce/edit/trunk/docs/release/rc.md",
+          "hash": "abf4055d5e4f0c576d23c4811d976f8abcdf51f3c961bbef582f18c9e2399569",
+          "url": "https://raw.githubusercontent.com/woocommerce/woocommerce/trunk/docs/release/rc.md",
+          "id": "db6ebbc37ebc0601b080ebf50d3ac5678403c198"
+        }
+      ],
+      "categories": []
+    },
```

Comment:
My understanding is that the manifest won't be required after the switch to Docusaurus (https://github.com/woocommerce/woocommerce/pull/57632). Just mentioning this as we probably want to hold off on merging this until that move is finalized.

Reply:
Yep, I synched the changes with trunk and the manifest nomore.